package.json

```diff
@@ -7,7 +7,7 @@
     "frontend",
     "backend",
     "dti-website",
-    "dti-website-server"
+    "image-cli"
```

Comment:
`dti-website-server` doesn't exist anymore. It's added by Jagger at some point during his deployment experimentation.